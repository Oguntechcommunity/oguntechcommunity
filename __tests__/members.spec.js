import * as React from "react";
import { mount } from "enzyme";
import Members from "../pages/members.js";

describe("Pages", () => {
	describe("Members", () => {
		it("should match snapshot", () => {
			const wrap = mount(<Members />);
			expect(wrap).toMatchSnapshot();
		});
	});
});
