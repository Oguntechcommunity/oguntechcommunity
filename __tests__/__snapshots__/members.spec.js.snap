// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages Members should match snapshot 1`] = `ReactWrapper {}`;
